set.mm

```diff
@@ -820087,6 +820224,463 @@ provide a criterion for two graphs being not isomorphic (see ~ grimgrtri ).
   $}
 
 
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Star graphs
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+
+   According to Wikipedia "Star (graph theory)", 10-Sep-2025,
+   ~ https://en.wikipedia.org/wiki/Star_(graph_theory) : "In graph theory, the
+   star S_k is the complete bipartite graph K(1,k), that is, it is a tree with
+   one internal node and k leaves.  Alternatively, some authors define S_k to
+   be the tree of order k with maximum diameter 2, in which case a star of
+   k > 2 has k - 1 leaves.".
+
+$)
+
+  $c StarGr $.
+
+  $( Extend class notation with star graphs. $)
+  cstgr $a class StarGr $.
+
+  ${
+    $d e n x $.
+    $( Definition of star graphs according to the first definition in
+       Wikipedia, so that ` ( StarGr `` N ) ` has size ` N ` , and order
+       ` N + 1 ` : ` ( StarGr `` 0 ) ` will be a single vertex (graph without
+       edges), see ~ stgr0 , ` ( StarGr `` 1 ) ` will be a single edge (graph
+       with two vertices connected by an edge), see ~ stgr1 , and
+       ` ( StarGr `` 3 ) ` will be a 3-star or "claw" (a star with 3 edges).
+       (Contributed by AV, 10-Sep-2025.) $)
+    df-stgr $a |- StarGr = ( n e. NN0
+                       |-> { <. ( Base ` ndx ) , ( 0 ... n ) >. ,
+                             <. ( .ef ` ndx ) , ( _I |` { e e. ~P ( 0 ... n ) |
+                                E. x e. ( 1 ... n ) e = { 0 , x } } ) >. } ) $.
+  $}
+
+  ${
+    $d N e n x $.
+    $( The star graph S(N).  (Contributed by AV, 10-Sep-2025.) $)
```

Comment:
Why not `S_N` rather than `S(N)` ? That's the standard notation for star graphs.

Reply:
> Why not `S_N` rather than `S(N)` ? That's the standard notation for star graphs.

You are right, I will change this.